rvsw/irve_rvsw_config.cmake

```diff
@@ -15,6 +15,7 @@ set(RVSW_DTB_PATH "${PROJECT_SOURCE_DIR}/../irve.dtb")
 set(RVSW_COMMON_FLAGS "-fomit-frame-pointer -Wall -Wextra -Werror -nostartfiles -static -static-libgcc --specs=nosys.specs -mstrict-align")                          
 set(RVSW_COMMON_FLAGS_DEBUG "-O0 -ggdb3")
 set(RVSW_COMMON_FLAGS_RELEASE "-O3 -flto=auto -fuse-linker-plugin")
+#set(RVSW_COMMON_FLAGS_RELEASE "-DNDEBUG -O3 -flto=auto -fuse-linker-plugin")#TODO eventually use this
```

Comment:
One step ahead of you! It's here: https://github.com/angry-goose-initiative/wiki/issues/41

I updated it to state that this PR resolves NDEBUG for IRVE but we still need to look into fixes for RVSW.

Reply:
Oh nice! In the future, it would be clearer if these TODO comments tell you what issue they are a part of (maybe by appending something like `(issue #41)` to the comment